### PR TITLE
Switch to new server

### PR DESCRIPTION
**Pull request by ChrisLaganiere**
Switch routes to point to the larger-capacity SHOUTcast server to increase max concurrent listeners

### CODE REVIEW

File: react/frontpage/StreamBar.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ var Collapse = require('react-bootstrap').Collapse;
 var Slider = require('react-slick');
 
 var trackURL = "https://ws.audioscrobbler.com/2.0/?method=user.getRecentTracks&user=uclaradio&api_key=d3e63e89b35e60885c944fe9b7341b76&limit=10&format=json";
-var streamURL = "http://stream.uclaradio.com:8000/listen";
+var streamURL = "http://uclaradio.com:8000/;";
```
Comment (MatteoVH) [inline react/frontpage/StreamBar.jsx]: what does the semicolon mean in the context of a URI?

Comment (hsuregan) [inline react/frontpage/StreamBar.jsx]: @ChrisLaganiere !

Comment (ChrisLaganiere) [inline react/frontpage/StreamBar.jsx]: yeah it's a bypass mechanism for SHOUTcast apparently! strange..

http://stackoverflow.com/questions/16871553/how-to-get-html5-to-play-a-stream-from-shoutcast

**Comment by ChrisLaganiere** [comment]
Yeah! @MatteoVH gonna run shoutcast on the digital ocean box, what do you think? It has 1TB / month bandwith. The mac mini seems pretty limited, tyler wennstrom checked it out and said another server might be better

**Comment by MatteoVH** [comment]
sure, that's awesome! i was just hearing the new app is great ;)

**Pull request merged**